posts/modular-monolith-microservices.md

```diff
@@ -8,29 +8,27 @@ title: "Why Companies are Moving their Applications to Modular Architecture"
 image:
 ---
 
-As applications become increasingly complex, engineering teams have sought to ease the process of creating scalable and malleable products. Many organizations have turned to microservices to help keep their products scalable. Today we will discuss why microservices & modular architecture have gained much attention and usage in the past few years.
+As applications become increasingly complex, engineering teams have sought to ease the process of creating scalable and malleable products. Many organizations have turned to microservices to help keep their products scalable, cost-efficient, and easier to develop over time. Today we will discuss why microservices & modular architecture have gained much attention and usage in the past few years.
 
 ## Monolithic Architecture
 
 To understand microservice architecture better, let's take a look into what traditional application architecture looks like - the monolith. "A monolithic architecture is a singular, large computing network with one code base that couples all of the business concerns together. To make a change to this sort of application requires updating the entire stack by accessing the code base and building and deploying an updated version of the service-side interface." [Atlassian](https://www.atlassian.com/microservices/microservices-architecture/microservices-vs-monolith#:~:text=A%20monolithic%20architecture%20is%20a%20singular%2C%20large%20computing%20network%20with,of%20the%20service%2Dside%20interface.)
 
 Organizations and teams worldwide have approached application development with an agile mindset, hardening the process of developing a monolith. Making a single change to your application's server or client-side can become an extensive process, with having to roll out the entire application again. This resource-intensive activity can make CI/CD difficult, so application architects and developers have leaned into a different type of architecture - microservice architecture. We'll first take a look at the Modular Monolith and how it differs from a traditional monolith before moving into microservices.
 
-![Monolith Vs Microservice Architecture Diagram](../../2.%20Tag%20and%20Link%20to%20Orient%20Ideas/Resources/Files/mono-micro-diagram.png) ***Microservice vs Monolith Architecture***
+![Monolith Vs Microservice Architecture Diagram](../images/2022/12/mono-micro-diagram.png) ***Microservice vs Monolith Architecture***
 
 ## Modular Monolith
```

Comment:
Just a heads up, Matt Reed may mention this as well, but on my blog everywhere I use the word "Monolith" he suggested it be changed to "Monolithic"

Reply:
Good point, i'll look for places I could change it to monolithic